nevergrad/functions/multiobjective/hypervolume.py

```diff
@@ -0,0 +1,252 @@
+import numpy as np
+from typing import List, AnyStr, Iterator
```

Comment:
for consistency, lately I've been updated all files little by little from `from typing import ....` to `import typing as tp` because the import line became way too big sometimes